lib/forest_liana/collections/customer.rb

```diff
@@ -3,6 +3,8 @@ class Forest::Customer
 
   collection :Customer
 
+  action 'Boom test'
```

Comment:
Boom test ?

Reply:
my bad I was interrupted by Lunchr a did a quick test, will update this accordingly